org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/servicetunnel/HttpServiceTunnel.java

```diff
@@ -352,6 +367,32 @@ protected void checkAlreadyCancelled(ServiceTunnelRequest serviceRequest) {
     }
   }
 
+  /**
+   * Cancels the remote service operation on server side.
+   */
+  public void cancel(ServiceTunnelRequest serviceRequest) {
```

Comment:
Is this method still necessary? Who is calling this?

Reply:
Not neccessary anymore, see https://github.com/eclipse-scout/scout.rt/pull/1765